packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/network/ovirtproviderovn.py

```diff
@@ -514,20 +514,23 @@ def _generate_pki(self):
                     'extract': True,
                     'user': oengcommcons.SystemEnv.USER_ROOT,
                     'keepKey': False,
+                    'shortLife': False,
                 },
                 {
                     'name':
                         oenginecons.OvnFileLocations.OVIRT_PROVIDER_OVN_SDB,
                     'extract': True,
                     'user': oengcommcons.SystemEnv.USER_ROOT,
                     'keepKey': False,
+                    'shortLife': False,
                 },
                 {
                     'name':
                         oenginecons.OvnFileLocations.OVIRT_PROVIDER_OVN_HTTPS,
```

Comment:
I'm not sure about this one. @almusil ?

Reply:
This is for the provider REST API, so it should probably follow other HTTP services that we have in engine.

Reply:
who talks to it? if only java backend then it should matter. or is called from UI anywhere?

Reply:
That's general API that is called by oVirt engine, but because it is neutron compatible it can be called by OpenStack Ansible and SDK. I am not sure if any of those has some policies about certificate lifetime. The key point is that, it can be called by anyone.